<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ASUS_Vivobook" failures="1" tests="1" name="Error Validation Test" time="8.503" errors="0" timestamp="2025-06-19T12:19:37 NPT">
  <testcase classname="ecommerceTesting.TestCases.ErrorValidationTest" name="loginErrorvalidation" time="5.822">
    <failure type="java.lang.AssertionError" message="expected [The account sign-in was incorrect or your account is disabled temporarily. Please wait and try again later.] but found [Th account sign-in was incorrect or your account is disabled temporarily. Please wait and try again later.]">
      <![CDATA[java.lang.AssertionError: expected [The account sign-in was incorrect or your account is disabled temporarily. Please wait and try again later.] but found [Th account sign-in was incorrect or your account is disabled temporarily. Please wait and try again later.]
at ecommerceTesting.TestCases.ErrorValidationTest.loginErrorvalidation(ErrorValidationTest.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- loginErrorvalidation -->
</testsuite> <!-- Error Validation Test -->
